pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -574,3 +619,28 @@ func namespacedName(itemMeta metav1.Object) string {
 	}
 	return itemMeta.GetName()
 }
+
+func setDefaultValues(c *cluster) {
+	if c.Namespace == "" {
+		c.Namespace = defaultNamespace
+	}
+	if c.ConfigMap == "" {
+		c.ConfigMap = defaultConfigMap
+	}
+	if c.ConfigMapKey == "" {
+		c.ConfigMapKey = defaultConfigMapKey
+	}
+}
+
+func hasRootCluster(config *cluster) bool {
+	return config.Namespace != "" ||
+		config.ConfigMap != "" ||
+		config.ConfigMapKey != "" ||
+		config.KubeConfigFilePath != "" ||
+		config.WebhookLabel != "" ||
+		config.APIServiceLabel != ""
+}
```

Comment:
Since the struct is comparable, i think this is a little more forwards compatible to do the following:
```suggestion
func hasRootCluster(config *cluster) bool {
	return *config != cluster{}    
}
```